### ISSUE TITLE
Quick question about sending x-hasura-admin-secret to access grapql access point

### ISSUE THREAD

**Issue by mattjslaton** [Flutterando/hasura_connect#100]
I am looking through the code now, where would I inject (or add) "x-hasura-admin-secret", I just switched over my graphql to the latest version and want to keep a password on it. I am using your client as the go between the database and server, it's working out very nicely. So, I appreciate all the hard work you guys have done on the project. Any pointers would be very much welcomed. Regards, Matt

**Comment by mattjslaton** [comment]
I got it! I just extended the class and override headers map. It seems to be working fine now. Here is the code I used if anyone is lost.

`class HasuraConnectWithAuth extends HasuraConnect {
  HasuraConnectWithAuth(String url) : super(url);
  @override
  var headers = {
    'content-type': 'application/json',
    'x-hasura-admin-secret': 'put your admin pass here'
  };
}`
Then you use `var hasuraConnect = HasuraConnectWithAuth(url);` instead of  `var hasuraConnect = HasuraConnect(url);`

Of course it would be wise to put your admin secret password in an ENV variable. And only use this on your server to talk to graphql as the go between the client and graphql.
